pkg/util/kube/kube.go

```diff
@@ -167,6 +173,155 @@ func EnsureClusterRequirements(ctx context.Context, kubeFactory kube.Factory, na
 		return fmt.Errorf("%s: %v", requirementsValidationErrorMessage, specificErrorMessages)
 	}
 
+	err = runNetworkTest(ctx, kubeFactory, currentContext.URL)
```

Comment:
just pass in the url if you don't use anything else from `config.AuthContext`

Reply:
Please add reporting here